### ISSUE TITLE
doc: describe how to use the container

### ISSUE THREAD

**Issue by hansfbaier** [FPGAOL-CE/osstoolchain-docker-things#1]
Hi,

it would be nice to have an example of how to use the container,
ie. the command with which to build the demo project.

Best regards,
Hans

**Comment by regymm** [comment]
Yes, more instructions will be added, and Makefile with locally stored .bin chipdb file will also be added. 
I'll leave this open cos there's far from enough how-to documents at this moment.
